tests/unit/dojson/test_dojson_hep.py

```diff
@@ -2011,7 +2011,7 @@ def test_publication_info(marcxml_to_json, json_to_marc):
             json_to_marc['773'][0]['w'])
     assert (marcxml_to_json['publication_info'][0]['pubinfo_freetext'] ==
             json_to_marc['773'][0]['x'])
-    assert (marcxml_to_json['publication_info'][0]['isbn'] ==
+    assert (marcxml_to_json['publication_info'][0]['parent_isbn'] ==
```

Comment:
Is this really how this should be tested? There are two functions, here, `f:JSON->MARC` and `g:MARC->JSON`.
We want to test that these two functions are inverse of each other, i.e. doing `f` then `g` is the identity, and the same holds for `g` then `f`. So if we know that `some_marc` is equivalent to `some_json`, the tests should assert `f(some_json)=some_marc` and `g(some_marc)=some_json`. Here, the test checks that a piece of `f(some_json)` contains the same info as `g(some_marc)`. This would pass if both transformations were to write the constant `1234567890` in the parent_isbn/773__z field, which is obviously wrong.